examples/Bunch/include/simulation_defines/param/radiationConfig.param

```diff
@@ -166,13 +169,17 @@ namespace radiation
  * radWindowFunctionGauss
  * radWindowFunctionNone
  */
-namespace radWindowFunctionTriangle { }
-namespace radWindowFunctionHamming { }
-namespace radWindowFunctionTriplett { }
-namespace radWindowFunctionGauss { }
-namespace radWindowFunctionNone { }
 
-namespace radWindowFunction = radWindowFunctionNone;
+  #ifndef PARAM_RADWINDOW
+  #define PARAM_RADWINDOW radWindowFunctionNone
+  #endif
+  namespace radWindowFunctionTriangle { }
+  namespace radWindowFunctionHamming { }
+  namespace radWindowFunctionTriplett { }
+  namespace radWindowFunctionGauss { }
+  namespace radWindowFunctionNone { }
+
+  namespace radWindowFunction = radWindowFunctionNone;
```

Comment:
use `namespace radWindowFunction = PARAM_RADWINDOW;` to let your option take effect...